style.css

```diff
@@ -0,0 +1,106 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
+
+.wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+}
```

Comment:
- [OPTIONAL] Kindly make your mobile version responsive until the desktop screen size. :+1: :+1: :smiley: 

![image](https://user-images.githubusercontent.com/98363075/192105415-42fd2058-d6ca-44c6-a237-04f938280f77.png)